sdk/eventhubs/azure-messaging-eventhubs/README.md

```diff
@@ -305,9 +305,9 @@ class Program {
             .consumerGroup("<< CONSUMER GROUP NAME >>")
             .connectionString("<< EVENT HUB CONNECTION STRING >>")
             .checkpointStore(new InMemoryCheckpointStore())
-            .processEvent(partitionEvent -> {
-                System.out.println("Partition id = " + partitionEvent.getPartitionContext().getPartitionId() + " and "
-                    + "sequence number of event = " + partitionEvent.getEventData().getSequenceNumber());
+            .processEvent(eventContext -> {
+                System.out.println("Partition id = " + eventContext.getPartitionContext().getPartitionId() + " and "
```

Comment:
thats an odd concat of two constants. Could they live in the same one?  `+ " and sequence ..."`